snmp/tests/test_utils.py

```diff
@@ -71,13 +77,9 @@ def test_oid_parse_invalid(value):
         pytest.param(ObjectType(ObjectIdentity((1, 3, 6, 1, 2, 1, 0))), (1, 3, 6, 1, 2, 1, 0), id='object-type'),
     ],
 )
-def test_oid_from_unresolved_instance(value, expected_tuple):
+def test_oid_as_object_type(value, expected_tuple):
     # type: (Any, Tuple[int, ...]) -> None
     oid = OID(value)
-
-    with pytest.raises(UnresolvedOID):
-        oid.as_tuple()
```

Comment:
Removed this assertion since it's now tested via `test_oid_from_unresolved_symbol`. This test is more focused as a result.